test/pluginsTests.ts

```diff
@@ -52,15 +52,4 @@ describe("Plugins", () => {
         expect(page.contents).with.lengthOf(3);
         expect((page.contents[2] as any).tag).to.equal("othertag");
     });
-
```

Comment:
like tsdoc can't just parse a string, so we can't just use a hardcoded string like we were for the markdown and CSS tests.